.github/workflows/integration.yml

```diff
@@ -1,6 +1,9 @@
 name: CI
```

Comment:
I do agree we have to be consistent between api and web - but I don't like acronyms.